### PR TITLE
[AllBundles] Update sortablejs back to 1.8.1

### LINKED ISSUE #2873: [AdminBundle] Pagepart sorting does not work
In https://github.com/Kunstmaan/KunstmaanBundlesCMS/commit/5262bd1a74e047513cbf044dbb6448981e9d84c1 is sortablejs reverted from 1.8.1 to 1.5.1. When upgrade this back to 1.8.1 pagepart sorting is working again.

### PR DESCRIPTION
**Pull request by dannyvw**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | fixes #2873

### CODE REVIEW

**Review by ProfessorKuma — CHANGES_REQUESTED** [review]
Hi @, your PR needs some changes
* This PR seems to need a milestone of a patch release.

**Review by ProfessorKuma — APPROVED** [review]
Hi @, your PR passed all our requirements.

Thank you for contributing!

File: package.json [hunk 1]
```diff
@@ -65,7 +65,7 @@
     "plupload": "^2.3.6",
     "ramda": "^0.23.0",
     "select2": "^4.0.3",
-    "sortablejs": "1.5.1",
+    "sortablejs": "1.8.1",
```
Comment (acrobat) [inline package.json:68]:
```suggestion
    "sortablejs": "^1.8.1",
```

**Review by acrobat — CHANGES_REQUESTED** [review]
1 minor comment, otherwise this PR is ready to merged. I've this on a new install and sorting is again working correctly.

Also revert the package-lock.json changes, this makes merging/upmerging easier and it will be updated on release when all assets are generated

**Comment by dannyvw** [comment]
@acrobat Done!

**Review by acrobat — APPROVED** [review]

**Comment by acrobat** [comment]
Thanks @dannyvw!

**Pull request merged**